rust/operator/src/config.rs

```diff
@@ -40,6 +42,9 @@ pub fn build_bootstrap_conf() -> String {
     bootstrap.insert("java.arg.2", "-Xms1024m".to_string());
     bootstrap.insert("java.arg.3", "-Xmx1024m".to_string());
 
+    // Enable Remote Debugging
+    // bootstrap.insert("java.arg.debug", "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8000".to_string());
+
```

Comment:
Remove?

Reply:
This is enabled in the original file, so I included it and commented it out since we probably don't need it, but I wanted to have it there for completeness. I saw that that was done with other config options as well, so that's why I did it. But I can also remove it.